sdk/language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -837,11 +846,14 @@ describe("interfaces", () => {
       Asset.toInterface(Token, ifaceContract.contractId),
       { newOwner: BOB_PARTY },
     );
+    const assetTemplateId = templateIdWithPackageId(buildAndLint.Main.Asset)(
```

Comment:
The template id returned from JSON API and the ledger will be different from the one passed in, in that it will contain a package id, rather than a package name. We use the new `templateIdWithPackageId` function to generate the templateId we expect to receive.